examples/databricks/example_databricks_workflow.py

```diff
@@ -53,19 +53,23 @@
         databricks_conn_id=DATABRICKS_CONN_ID,
         job_clusters=job_cluster_spec,
         notebook_params=[],
+        notebook_packages=[
+            {"pypi": {"package": "simplejson"}},
+        ],
     )
     with task_group:
         notebook_1 = DatabricksNotebookOperator(
             task_id="notebook_1",
             databricks_conn_id=DATABRICKS_CONN_ID,
-            notebook_path="/Users/daniel@astronomer.io/Test workflow",
+            notebook_path="/Users/pankaj.koti@astronomer.io/Notebook1",
+            notebook_packages=[{"pypi": {"package": "Faker"}}],
             source="WORKSPACE",
             job_cluster_key="Shared_job_cluster",
         )
         notebook_2 = DatabricksNotebookOperator(
             task_id="notebook_2",
             databricks_conn_id=DATABRICKS_CONN_ID,
-            notebook_path="/Users/daniel@astronomer.io/Test workflow",
+            notebook_path="/Users/pankaj.koti@astronomer.io/Notebook2",
```

Comment:
is there a better way to do these local `notebook_paths`? maybe we can create a shared folder in dbr so we don't need to keep changing this?

Reply:
yes, using shared notebooks now. https://github.com/astronomer/astronomer-cosmos/pull/159/commits/e7c47f8773d194039ec432eb6226801e0b2793d3